docs/current_docs/agents.mdx

```diff
@@ -59,7 +59,13 @@ Once this feature is merged (current target is 0.17), a development build will n
 Install the development version of LLM-enabled Dagger:
 
 ```console
-curl -fsSL https://dl.dagger.io/dagger/install.sh | DAGGER_VERSION=0.17.0-llm.4 BIN_DIR=/usr/local/bin sh
+curl -fsSL https://dl.dagger.io/dagger/install.sh | DAGGER_VERSION=0.17.0-llm.4 BIN_DIR=$HOME/.local/bin sh
+```
+
+If you want to install globally, and sudo is available, you can specify an alternative install location by setting BIN_DIR and using sudo -E:
+
+```console
+curl -fsSL https://dl.dagger.io/dagger/install.sh | DAGGER_VERSION=0.17.0-llm.4 BIN_DIR=/usr/local/bin SUDO -E sh
```

Comment:
though, imo this can be omitted, I think it's obvious enough from seeing `BIN_DIR` above, plus the sentence below ("You can adjust BIN_DIR ..."):

```suggestion
```

piping to `sudo bash` spooks some folks out so maybe it's best to leave that exercise to the reader 😆